docs/src/refs.bib

```diff
@@ -165,6 +165,8 @@ @article{GoerzJPB2011
     Doi = {10.1088/0953-4075/44/15/154011},
     Pages = {154011},
     Volume = {44},
+    eprint = {1103.6050},
```

Comment:
I have a question on this one – how do you know this is an arXiv preprint?  Or is that the default? In BibLaTeX there is `EPRINTTYPE` for that, see, e.g. https://github.com/JuliaManifolds/Manopt.jl/blob/b574950cf349013232e8ab07fa355046bd90e8d3/docs/src/references.bib#L504-L517, and I already have a few that are HAL ( https://hal.science/?lang=en, mainly French authors) which would require a different url prefix.

Reply:
The field for that is `Archiveprefix`. It's supposed to be an alias for `eprinttype` (cf. the biblatex manual). I don't think Bibliography.jl understands `eprinttype`, though (but it should, so that would be something to open an issue on; also `primaryclass`/`eprintclass`). [ArXiv itself recommends `Archiveprefix`](https://info.arxiv.org/help/hypertex/bibstyles/index.html).

The one thing I did was make `Archiveprefix={arxiv}` the default ([which you can also do in LaTeX](https://tex.stackexchange.com/questions/386078/workaround-for-missing-archiveprefix-in-bib-entry)). See the notes on the [Syntax for the `.bib` file](https://juliadocs.org/DocumenterCitations.jl/previews/PR32/syntax/#Syntax-for-the-.bib-file) in the PR.

Other Archiveprefixes (e.g., `Archiveprefix={bioRxiv}`) are supported and will render correctly, but currently aren't linked. I'm very open to adding support (i.e., automatic linking) for other preprint servers. All I need to know is the mapping from `eprint` to a URL for a given preprint server.

Actually, [bioRxiv](https://www.biorxiv.org) is the only other preprint server I was even aware of, and I couldn't find any LaTeX `.bst` file that has linking support for it (or any other preprint server besides arXiv). Not that that should prevent me from adding support, as soon as someone requests it.

So yeah, if you can open a new issue for HAL with a link to how HAL defines `eprint` identifiers and how that maps to a URL/DOI, I'll be happy to add linking support for it in another PR.

Reply:
Thanks for the clarification :) 

Sure we should link probably at least bioRxiv and HAL – I can check for the URL, since I have that somewhere in my LaTeX/BibLaTeX document header.